docs/security.rst

```diff
@@ -239,6 +248,28 @@ The following driver code specifies that the connection should use two-way verif
 The driver uses ``SSLContext`` directly to give you many other options in configuring SSL. Consider reading the `Python SSL documentation <https://docs.python.org/library/ssl.html#ssl.SSLContext>`_
 for more details about ``SSLContext`` configuration.
 
+**Server verifies client and client verifies server using Twisted and pyOpenSSL**
+
+.. code-block:: python
+
+    from OpenSSL import SSL, crypto
+    from cassandra.cluster import Cluster
+    from cassandra.io.twistedreactor import TwistedConnection
+
+    ssl_context = SSL.Context(SSL.TLSv1_METHOD)
+    ssl_context.set_verify(SSL.VERIFY_PEER, callback=lambda _1, _2, _3, _4, ok: ok)
```

Comment:
just add the `ok` import

Reply:
There is no import. It's just the 5th argument to the lambda.

Reply:
true :man_facepalming: